test/itemlistformaction.cpp

```diff
@@ -133,6 +133,40 @@ TEST_CASE(
 	REQUIRE(feed->unread_item_count() == 0);
 }
 
+TEST_CASE(
+	"OP_OPENBROWSER_AND_MARK does not mark read when browser fails",
+	"[ItemListFormAction]")
+{
+	Controller c;
+	newsboat::View v(&c);
+	TestHelpers::TempFile browserfile;
```

Comment:
You can remove this object since the "mock browser" (i.e. `false %u`) doesn't write anything there.